README.md

```diff
@@ -1,39 +1,39 @@
-# Ubuntu 22.04 based MLflow image v2.1.1
```

Comment:
Could we also add the badges for the CI, similarly to 
https://github.com/canonical/training-operator-rock/blob/main/README.md